src/display/annotation_layer.js

```diff
@@ -451,7 +451,11 @@ class TextWidgetAnnotationElement extends WidgetAnnotationElement {
       // NOTE: We cannot set the values using `element.value` below, since it
       //       prevents the AnnotationLayer rasterizer in `test/driver.js`
       //       from parsing the elements correctly for the reference tests.
-      const textContent = storage.getOrCreateValue(id, this.data.fieldValue);
+      const textContent = storage.getOrCreateValue(
+        id,
+        this.data.fieldValue,
+        true
```

Comment:
Calling a function/method with an explicit boolean parameter like this makes it more difficult to know immediately, when reading the code, what the parameter actually represents without looking up the definition.

Usually, to aid readability in these cases, adding an inline comment will help. Hence changing this line, and similar lines below, to the following would help: 
```suggestion
        /* isInitialValue = */ true
```

Reply:
On top of that, I this such a parameter is not really necessary. I think we can avoid touching _any_ callsite here by tracking this internally in the `AnnotationStorage` class. The first time the default values are stored in the storage, so any second change means it's modified.